syft/frameworks/torch/nn/functional.py

```diff
@@ -20,10 +20,10 @@ def dropout(input, p=0.5, training=True, inplace=False):
         binomial = torch.distributions.binomial.Binomial(probs=1 - p)
 
         # we must convert the normal tensor to fixed precision before multiplication
-        # Note that: Weights of a model are alwasy Float values
+        # Note that: Weights of a model are always Float values
         # Hence input will always be of type FixedPrecisionTensor > ...
         noise = (binomial.sample(input.shape).type(torch.FloatTensor) * (1.0 / (1.0 - p))).fix_prec(
-            **input.get_class_attributes(), no_wrap=True
+            no_wrap=True
```

Comment:
@LaRiffle The reason I removed it was that `input.get_class_attributes()` returned a dictionary `{"type": torch.float32}` and in turn is passed to syft.FixedPrecisionTensor as kwargs inside fix_prec function. But since FixedPrecisionTensor doesn't have any variable 'type' it throws error unexpected key argument.